test/chartmogul/opportunity.js

```diff
@@ -0,0 +1,174 @@
+'use strict';
+
+const ChartMogul = require('../../lib/chartmogul');
+const config = new ChartMogul.Config('token');
+const expect = require('chai').expect;
+const nock = require('nock');
+const Opportunity = ChartMogul.Opportunity;
+
+describe('Opportunity', () => {
+  it('creates a opportunity for a customer', async () => {
+    const uuid = 'cus_00000000-0000-0000-0000-000000000000';
+    const postBody = {
+      customer_uuid: uuid,
+      owner: 'test1@example.org',
+      pipeline: 'New business 1',
+      pipeline_stage: 'Discovery',
+      estimated_close_date: '2023-12-22',
+      currency: 'USD',
+      amount_in_cents: 100,
+      type: 'recurring',
+      forecast_category: 'pipeline',
+      win_likelihood: 3,
+      custom: [{ key: 'from_campain', value: 'true' }]
```

Comment:
[NON-BLOCKING]

Also the same here as with the file above, does not affect the logic as it is all custom, but the word `campain` should be spelled `campaign` here and several other places in this file.